scripts/smoke_test.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+set -euxo pipefail
+
+# provide a default environment if none exists
+if [ ! -f ".env" ]; then
+    echo "Using .env.oss"
+    cp .env.oss .env
+fi
+
+# prepare a production build
+yarn assets
+yarn build:server
+
+# start server in the background in production mode
+NODE_ENV=production yarn start &
+
+# wait for it to accept connections
+yarn run wait-on http://localhost:5000
+
+# run ./cypress/integration/* tests in headless mode
+./node_modules/.bin/cypress install
```

Comment:
We changed this syntax while debugging, but now it is different from the above `wait-on` line-- I'm ok with either option but should be consistent.

Reply:
Good call, will fix